constrained-voucher.mkd

```diff
@@ -147,10 +147,7 @@ This document specifies a constrained voucher-request artifact based on {{Sectio
 voucher(-request) transport over CoAP based on {{Section 3 of RFC8995}} and on {{I-D.ietf-ace-coap-est}}.
 
 The CBOR definitions for the constrained voucher format are defined using the mechanism described in {{I-D.ietf-core-yang-cbor}} using the SID mechanism explained in {{I-D.ietf-core-sid}}.
-As the tooling to convert YANG documents into a list of SID keys is still in its infancy, the table of SID values presented here MUST be considered normative rather than the output of the pyang tool.
-
-There is additional work when the voucher is integrated into the key-exchange, described in {{!I-D.selander-ace-ake-authz}}.
-This work is not in scope for this document.
+As the tooling to convert YANG documents into a list of SID keys is still in its infancy, the table of SID values presented here MUST be considered normative rather than the output of the tool specified in {{I-D.ietf-core-sid}}.
```

Comment:
Why remove the statement about ACE-AKE-AUTHZ as being out of scope?

Reply:
It is a new subject. Not saying anything means out of scope.